gui/src/components/s3/account-management.vue

```diff
@@ -468,6 +477,122 @@
         </cortx-has-access>
       </v-col>
     </v-row>
+    
+    <v-dialog 
+      v-model="showResetAccountForm" 
+      persistent
+      max-width="500" 
+      id="s3-resetaccount-form">
+      <v-card>
+        <v-card-title class="title mt-6 ml-3">
+          <span>{{ $t("s3.account.reset-password") }}</span>
```

Comment:
"s3.account.reset-password" why "-" is used in reset-password. For variables we need to follow the standards. If its coming from REST API, this have to be "_" and not -.

Reply:
It's from en.json